### ISSUE TITLE
wrong model folders 

### ISSUE THREAD

**Issue by harshitwalia** [shamangary/SSR-Net#4]
error in line 76 to 81 in master/training_and_testing/SSRNET_train.py
 if db_name == "wiki":
        weight_file = "wiki_models/"+save_name+"/"+save_name+".h5"
        model.load_weights(weight_file)
 elif db_name == "imdb": 
        weight_file = "imdb_models/"+save_name+"/"+save_name+".h5"
        model.load_weights(weight_file) 
elif db_name == "morph": 
        weight_file = "morph_models/"+save_name+"/"+save_name+".h5"
        model.load_weights(weight_file)

**Comment by shamangary** [comment]
Hello. It is not wrong. 

As we point out in our paper section 4.3,
"Since the IMDB dataset is much larger, we trained our model on IMDB first and used the trained model as the starting point for training on the WIKI dataset."

In our section 4.4, "Before training with MORPH2, we used the IMDB-WIKI dataset for pre-training."

When we want to train morph2, we use the trained model from WIKI as a starting point for the new training.
